package.json

```diff
@@ -4,10 +4,22 @@
   "dependencies": {
     "@hotwired/stimulus": "^3.2.1",
     "@hotwired/turbo-rails": "^7.3.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "webpack": "^5.85.1",
     "webpack-cli": "^5.1.3"
   },
   "scripts": {
     "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.22.1",
+    "@babel/preset-env": "^7.22.4",
+    "@babel/preset-react": "^7.22.3",
+    "babel-loader": "^9.1.2",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
```

Comment:
- [ ] When I try to precompile the assets this error is raised:

![image](https://github.com/techEmprez/hello_rails_react/assets/68796232/34cae9ae-8280-4bd8-8c85-ef82eb29ed6c)

Then when I try to run the rails server to test your app this error is raised:

![image](https://github.com/techEmprez/hello_rails_react/assets/68796232/c8524047-b8a7-498b-847d-c9e29ef6eeae)

That's happening because you used assets and libraries in your react app and didn't add it to the `package.json` file so when webpack tries to compile the code it raised the 8 error messages in the first images.
Kindly add those libraries to the `package.json` file (`react-redux` `react-router-dom` `@reduxjs/toolkit` `redux-logger` `redux-thunk`), 
After I installed those packages, run `rails assets:precompile`, and then run the rails server everything works as expected

Reply:
All dependencies added and functional
For any enquiry or changes, Please, reviewer can you reach out via zoom or slack as i am available.

Thank you.